crates/wit-component/src/linking/metadata.rs

```diff
@@ -131,7 +131,7 @@ pub struct Import<'a> {
     pub module: &'a str,
     pub name: &'a str,
     pub ty: Type,
-    pub flags: u32,
+    pub flags: SymbolFlags,
```

Comment:
I'm not familiar with the component model, is there a reason we need to use `BTreeSet` to keep imports & exports sorted? It's indirectly requiring `SymbolFlags` to also derive `Ord`, which I'd prefer to avoid as it doesn't make sense semantically.

Reply:
Yes and no, it depends on the location, but it's probably easiest to just leave `Ord` in there